plugins/woocommerce/tests/api-core-tests/README.md

```diff
@@ -127,25 +129,279 @@ After you run a test, it's best to restart the environment to start from a fresh
+
+The [Playwright documentation](https://playwright.dev/docs/intro) is a good source for finding out more details on the various methods used in the tests, including what is available to you when you write new tests. The [API testing](https://playwright.dev/docs/test-api-testing) section has a good example on how the [playwright.config.js](https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce/tests/api-core-tests/playwright.config.js) is used (see [Configuration](https://playwright.dev/docs/test-api-testing#configuration) section) and also how [Setup and Teardown](https://playwright.dev/docs/test-api-testing#setup-and-teardown) works. 
+
+Note: Playwright uses the [expect](https://jestjs.io/docs/expect) library for test assertions. This library provides a lot of matchers like `toEqual`, `toContain`, `toHaveLength` and many more. Examples of these are throughout the test files.
+
+## What aspects of the API should we test?
+
+Assuming that we have validated the API contract (inspected the spec/contract, made sure the endpoints are correctly named, resources and types reflect the object model and there is no missing/duplicate functionality) we are ready to test. 
+
+
+For each API request method (`GET`, `POST`, `PUT`, `DELETE`, `PATCH`), the test would need to take the following actions: 
+1. Verify correct HTTP status code. For example, creating a resource should return 201 CREATED and unpermitted requests should return 403 FORBIDDEN, etc.
+2. Verify response payload. Check valid JSON body and correct field names, types, and values — including error responses.
+3. Verify response headers where appropriate e.g. Some headers hold information related to search totals, pagination values etc. (see [Examples](#examples) below).
+
+At a minimum, we want to ensure each possible CRUD operation can be applied and sufficient assertions have been validated based on the above. 
+
+A reasonable process would be:
+1. Create an object using a `POST` request
+2. Retrieve the created object using the `GET` request
+3. Update the object using a `POST` request
+4. Delete the object using a `DELETE`request
+5. Test any batch create/update/delete operations (if applicible)
+
+Additional tests can also be added to test the following general test scenarios:
+
+- Basic positive tests (happy paths) - check basic functionality and the acceptance criteria of the API
+- Extended positive testing with optional parameters - more thorough testing (can be used for testing a bug/updates/new functionality)
+- Negative testing with valid input - expect to gracefully handle problem scenarios with valid user input e.g. trying to add an existing username
+- Negative testing with invalid input - expect to gracefully handle problem scenarios with invalid user input e.g. trying to add a username which is null
```

Comment:
Should we maybe explain that with negative testing we intent to validate that we get error responses when expected as per specification and the error status code and message are correct as per documentation?